apps/crn-server/test/data-providers/contentful/users.data-provider.test.ts

```diff
@@ -576,6 +582,16 @@ describe('User data provider', () => {
         },
       });
     });
+
+    test('sets `createdDate` when a user is onboarded', async () => {
```

Comment:
what if they were already onboarded? shouldn't that be tested and handled?

Reply:
Not sure I follow. Anyone already onboarded would be the users migrated from Squidex, which have this data point copied over. This is for new people joining the hub in future.

There is the edge case for people who have accounts created manually in the CMS with onboarding skipped, but for those the fallback to `firstPublishedAt` would suffice. Do you mean I should add a test case for that fallback?